cabal-testsuite/cabal-testsuite.cabal

```diff
@@ -115,6 +115,6 @@ custom-setup
   -- and due to Custom complexity and ConstraintSetupCabalMaxVersion
   -- it has to be the latest release version plus
   -- you have to use the latest cabal-install release
-  setup-depends: Cabal == 3.8.*,
-                 Cabal-syntax == 3.8.*,
+  setup-depends: Cabal ^>= 3.10,
```

Comment:
So, shall we try reverting this to 3.8? This is completely dark art and only affects custom setup tests, so whatever works IMHO.